services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/DigitalGreenCertificateStructureProvider.java

```diff
@@ -69,15 +70,19 @@ private DirectoryOnDisk constructArchiveToPublish(DigitalGreenCertificate dgcCon
     DirectoryOnDisk dgcDirectory = new DirectoryOnDisk(dgcConfig.getDgcDirectory());
 
     for (String currentLanguage: dgcConfig.getSupportedLanguages()) {
-      ArchiveOnDisk archiveToPublish = new ArchiveOnDisk(dgcConfig.getValuesetsFileName());
-      archiveToPublish.addWritable(new FileOnDisk(EXPORT_BIN,
-          dgcToProtobufMapping.constructProtobufMapping().toByteArray()));
-      DirectoryOnDisk languageDirectory = new DirectoryOnDisk(currentLanguage.toLowerCase());
-      languageDirectory.addWritable(new DistributionArchiveSigningDecorator(
-          archiveToPublish, cryptoProvider, distributionServiceConfig));
-      dgcDirectory.addWritable(languageDirectory);
-      logger.info("Writing digital green certificate to {}/{}/{}.", dgcDirectory.getName(), languageDirectory.getName(),
-          archiveToPublish.getName());
+      try {
```

Comment:
Is the try/catch inside the for-loop on purpose?

Reply:
with this, at least there is a chance that some languages don't fail :-)